processor/src/processor/operations/u32_ops.rs

```diff
@@ -32,7 +32,7 @@ macro_rules! require_u32_operands {
                 return Err(ExecutionError::not_u32_values(invalid_values, $errno, $err_ctx));
             }
             // Return tuple of operands based on indices
-            ($([<operand_ $idx>].as_int()),*)
+            ($([<operand_ $idx>]),*)
```

Comment:
Ideally this would return the `u64` to avoid converting them again. But let's do this in a different PR.